startup/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ruby:2.6.6-alpine3.12
```

Comment:
Rename this to `Dockerfile.dev` pls.

(We're going to rename them all to `development.Dockerfile` as that's the official way, so you might want to start that trend here instead).